bin/p2-exec/main.go

```diff
@@ -0,0 +1,163 @@
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"syscall"
+
+	"github.com/square/p2/pkg/cgroups"
+	"github.com/square/p2/pkg/util"
+	"gopkg.in/yaml.v2"
+)
+
+// usage without cgroups:
+//   p2-exec username envdir launchable:cgname myexecutable myargs
+// there must be a PLATFORM_CONFIG_PATH envar (possibly in the envdir) to
+// specify the cgroup parameters for the <launchable>, which is created under
+// <cgname>
+func main() {
+	username := os.Args[1]
```

Comment:
Reason for straying from kingpin? It supports args and I'd like to see a usage when executed, even if this is executed automatically most of the time.